webhook/webhook.go

```diff
@@ -126,6 +127,7 @@ type AdmissionController struct {
 type GenericCRD interface {
 	apis.Defaultable
 	apis.Validatable
+	apis.Annotatable
```

Comment:
I wouldn't require this, I'd check this like we do for `apis.Immutable`

Reply:
Done.